asserts/ifacedecls.go

```diff
@@ -148,16 +150,25 @@ func (matcher mapAttrMatcher) feature(flabel string) bool {
 	return false
 }
 
-func (matcher mapAttrMatcher) match(apath string, v interface{}, ctx AttrMatchContext) error {
+func (matcher mapAttrMatcher) match(attrer interfaces.Attrer, apath string, v interface{}, ctx AttrMatchContext) error {
+	if apath == "" && attrer != nil {
```

Comment:
What is going on here? Could you add a comment please?

Reply:
Added a comment and also simplified the conditional.